assets/js/blocks/cart-checkout/mini-cart-contents/inner-blocks/mini-cart-products-table-block/index.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { Icon, list } from '@woocommerce/icons';
+import { registerFeaturePluginBlockType } from '@woocommerce/block-settings';
+
+/**
+ * Internal dependencies
+ */
+import { Edit } from './edit';
+import metadata from './block.json';
+
+registerFeaturePluginBlockType( metadata, {
+	icon: {
+		src: (
+			<Icon
+				srcElement={ list }
+				className="wc-block-editor-components-block-icon"
+			/>
+		),
+	},
```

Comment:
Nitpick, but just so you know, this can be rewritten as:
```suggestion
	icon: (
		<Icon
			srcElement={ list }
			className="wc-block-editor-components-block-icon"
		/>
	)
```